integration-tests/bats/sql-server-config.bats

```diff
@@ -54,6 +54,18 @@ teardown() {
     [[ "$output" =~ "sqlserver.global.max_connections = 1000" ]] || false
 }
 
+@test "sql-server-config: dolt_transaction_commit is global variable" {
+    cd repo1
+    start_sql_server repo1
+
+    insert_query repo1 1 "SET @@PERSIST.dolt_transaction_commit = 1"
```

Comment:
This only really tests the PERSIST aspect of it. To see if it's GLOBAL you should add a test in sql-server.bats that makes sure that the value persists across sessions.